### ISSUE TITLE
Value List Item (advanced Panel) - Provide a valid label for form fields - (2036570517)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#5634]
### Module: 
 42 Calcite Panel 

### Violation: 
 Provide a valid label for form fields 

![image](https://user-images.githubusercontent.com/5023024/202035692-464ee890-0ae7-464c-b4c1-0be7c0e81a99.png)

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570517 

### Severity: 
 10 

### Description: 
 [Issue]
There are buttons without accessible names. Examples include:

- Events
- Data
- Documents
- Apps & Maps
- Custom Catalog 1
- Initiatives

[User Impact]
Screen reader users will be unable to determine the purpose of these buttons. Speech input users will have difficulty activating them.

[Code Reference] 
 ```
<span class="handle" data-js-handle="" role="button" tabindex="0">
  <div aria-hidden="true" icon="drag" scale="s" calcite-hydrated="">
    (...)
  </div>
</span>
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with an aria-label attribute. Good accessible names are both concise and descriptive.

[Compliant Code Example] 
 ```
<span aria-label="Events, press space and use arrow keys to reorder content. Current position 1 of 5." class="handle" data-js-handle="" role="button" tabindex="0">
  <div aria-hidden="true" icon="drag" scale="s" calcite-hydrated="">
    (...)
  </div>
</span>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Pertains to the `value-list-item` component's `drag-enabled`.

![image](https://user-images.githubusercontent.com/5023024/202035692-464ee890-0ae7-464c-b4c1-0be7c0e81a99.png)

**Comment by driskull** [comment]
I think we can defer this one for now.

**Comment by geospatialem** [comment]
The handle button still does not provide context in `1.4.2`. But considered a low priority with the deprecation. Leaving without a milestone for now.

**Comment by geospatialem** [comment]
Adding to the freezer milestone due to the deprecated component.

**Comment by geospatialem** [comment]
Closing in favor of https://github.com/Esri/calcite-design-system/issues/7426
